Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -45,9 +54,9 @@ extension SentryFeedback: SentrySerializable {
             dict["contact_email"] = email
         }
         if let associatedEventId = associatedEventId {
-            dict["associated_event_id"] = associatedEventId
+            dict["associated_event_id"] = associatedEventId.sentryIdString
         }
-        dict["source"] = source.rawValue
+        dict["source"] = source.description
```

Comment:
`l`: See my comment in #4874

https://github.com/getsentry/sentry-cocoa/pull/4874#discussion_r1965182405

Reply:
Took the suggestion on this one, see my comments there. Thanks!